src/main/aws/sso.ts

```
@@ -0,0 +1,147 @@
+        height: 800,
+        title: "AWS SSO Login",
+        autoHideMenuBar: true,
+        webPreferences: {
+          nodeIntegration: false,
+          contextIsolation: true,
+        },
+      });
+
+      // Load the verification URL
+      console.log("Loading URL:", deviceAuthResponse.verificationUriComplete);
+      await authWindow.loadURL(deviceAuthResponse.verificationUriComplete!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)